app/controllers/EventReportController.scala

```diff
@@ -64,6 +64,23 @@ class EventReportController @Inject()(
       }
   }
 
+  def compileEventOneReport: Action[AnyContent] = Action.async {
+    implicit request =>
+      post { (pstr, userAnswersJson) =>
+        logger.debug(message = s"[Compile Event 1 Report: Incoming-Payload]$userAnswersJson")
```

Comment:
better if you remove the commented code until it is ready.

Reply:
This was left unimplemented as validation was part of 7145, now implemented: https://github.com/hmrc/pension-scheme-event-reporting/pull/11/files#diff-1e7681c64f3d16a12b604f6fad9a7e98cb624f0cd37af940a018028ee6480893

Reply:
I though we are doing TDD ;). writing the test before implementation , I will resolve it it for now, but please in future we should be restricted within the scope of the ticket. thanks